### DISCUSSION TITLE
Tooltip作为受控组件使用的demo

### DISCUSSION THREAD

**Discussion by mrdulin** [ant-design/ant-design#51976] [Q&A]
```jsx
import React, { useState } from "react";
import "./index.css";
import { Tooltip } from "antd";

const App: React.FC = () => {
  const [open, setOpen] = useState(false);
  return (
    <Tooltip
      title={
        <div>
          <p>Tooltip will show on mouse enter.</p>
          <p>Tooltip will show on mouse enter.</p>
          <p>Tooltip will show on mouse enter.</p>
          <p>Tooltip will show on mouse enter.</p>
          <p>Tooltip will show on mouse enter.</p>
        </div>
      }
      open={open}
    >
      <span
        onMouseEnter={() => setOpen(true)}
        onMouseLeave={() => setOpen(false)}
      >
        show tooltip
      </span>
    </Tooltip>
  );
};

export default App;
```

问题：调整窗口尺寸，让tooltip弹出后悬浮在触发的`span`元素上，弹出tooltip后，会触发`span`的`onMouseLeave`，`open` state被设置为false, tooltip隐藏后，再次触发`span`的`onMouseEnter`，如此反复，这种情况应该怎么处理

![GIF 2024-12-11 16-35-51](https://github.com/user-attachments/assets/9d09c0ad-3bef-4381-8920-beab7ec200d6)

[codesandbox](https://codesandbox.io/p/sandbox/basic-antd-5-22-4-forked-8zp9y8)

**Comment by Wxh16144** [comment]
把 `onMouseLeave` 放在这里是不是就可以了～
<img width="719" alt="image" src="https://github.com/user-attachments/assets/4a1efbd9-f3f7-4900-ba34-d2fd2e27b556">

**Reply by mrdulin** [reply]
鼠标移动到tooltip右侧空白处，tooltip不会消失，和非受控的行为不一致哦
